cpp/Makefile

```diff
@@ -240,15 +291,19 @@ directories:
 	mkdir -p $(BUILD_DIR)
 	cp ../FAudio.dll $(BUILD_DIR)
 	cp `sdl2-config --prefix`/bin/SDL2.dll $(BUILD_DIR)
+	cp `sdl2-config --prefix`/bin/libwinpthread*.dll $(BUILD_DIR)
 	if [ -n "${FAUDIO_FFMPEG}" ] && [ -n "${FAUDIO_FFMPEG_PREFIX}" ]; then	\
 		cp $(FAUDIO_FFMPEG_PREFIX)/bin/avcodec*.dll $(BUILD_DIR);	\
 		cp $(FAUDIO_FFMPEG_PREFIX)/bin/avutil*.dll $(BUILD_DIR);	\
+		cp $(FAUDIO_FFMPEG_PREFIX)/bin/swresample*.dll $(BUILD_DIR);	\
 	fi
 
 clean:
 	rm -f $(XAUDIO20_TARGET) $(XAUDIO21_TARGET) $(XAUDIO22_TARGET) $(XAUDIO23_TARGET) \
 	      $(XAUDIO24_TARGET) $(XAUDIO25_TARGET) $(XAUDIO26_TARGET) $(XAUDIO27_TARGET) \
-	      $(XAUDIO28_TARGET) $(XAUDIO29_TARGET) $(X3DAUDIO_TARGET) $(XAPO_TARGET) \ \
+	      $(XAUDIO28_TARGET) $(XAUDIO29_TARGET) $(X3DAUDIO_TARGET) $(XAPO_TARGET) \
+              $(XAPOFX11_TARGET) $(XAPOFX12_TARGET) $(XAPOFX13_TARGET) \
```

Comment:
Extra couple spaces?

Reply:
I dunno if it's GitHub's spacing or what but for some reason these two lines have two extra spaces on them, and I dunno if it's the site or the actual file...

Reply:
Oh you know what it is, we have mixed tabs/spaces here, when the crap did that happen...

This one's not your fault, will deal with this later!